pages/api/services/[id].js

```diff
@@ -0,0 +1,16 @@
+import dbConnect from "../../../db/connect";
+import Service from "../../../db/models/Service";
+
+export default async function handler(request, response) {
+  await dbConnect();
+  const { id } = request.query;
+
+  if (request.method === "GET") {
+    const service = await Service.findById(id);
+
+    if (!service) {
+      return response.status(404).json({ status: "not found" });
+    }
+    response.status(200).json(service);
+  }
+}
```

Comment:
**This endpoint doesn't seem to be directly relevant to the current user story.** 

For the following pull requests, please keep the following in mind: 🥺
_It is important that we **focus on the current requirements** and ensure that the code we write is directly related to those requirements._